config/settings.py

```diff
@@ -69,8 +70,24 @@ def __init__(self, path: str):
             self.path = path
 
     class Server:
-        def __init__(self, user_ip: str, private_key: str, endpoint_ip: str, endpoint_port: str):
+        def __init__(self, user_ip: str, private_key: str, public_key:str, endpoint_ip: str, endpoint_port: str):
             self.user_ip = user_ip
             self.private_key = private_key
+            self.public_key = public_key
             self.endpoint_ip = endpoint_ip
             self.endpoint_port = endpoint_port
+
+        def wg_server_base_str(self):
+            return f"""[Interface]
+Address = {self.user_ip}.1/24
+ListenPort = {self.endpoint_port}
+PrivateKey = {self.private_key}
+"""
+
+        def create_wg_server_config(path, base):
+            with open(path, "w", encoding="utf-8") as wg_file:
+                wg_file.write(base)
+
+        def update_wg_server_config(path, data):
+            with open(path, "a", encoding="utf-8") as wg_file:
+                wg_file.write(data)
```

Comment:
what is `base` and `data`? it's pretty unclear, so it may be better to specify which data actually needed to be written

Reply:
Damn, I've thought that I deleted it